cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package staticpod
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/ghodss/yaml"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
```

Comment:
I know this was not net-new, but we should cut this dep and instead use `ioutil.WriteFile` below like this:
```go
if err := ioutil.WriteFile(filename, serializedBytes, 0700); err != nil {
     return fmt.Errorf("failed to write static pod manifest file for %q (%q) [%v]", name, filename, err)
}
```

As a consequence, you will be able to remove the bad dep here: https://github.com/kubernetes/kubernetes/blob/master/build/visible_to/BUILD#L77 :+1: